src/core/utils/humanize/smartSplit.ts

```diff
@@ -84,15 +90,22 @@ function* tokenize(source: string): Generator<SmartSplitToken> {
         switch (source[i]) {
             case ' ':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.BREAK, start: i, end: i + 1 };
+                yield { type: SmartSplitTokenType.BREAK, start: i, end: i + 1, content: ' ' };
                 break;
             case '"':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.QUOTE, start: i, end: i + 1 };
+                yield { type: SmartSplitTokenType.QUOTE, start: i, end: i + 1, content: '"' };
                 break;
             case '\\':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.ESCAPE, start: i, end: i + 1 };
+                yield {
+                    type: SmartSplitTokenType.LITERAL,
+                    start: i,
+                    end: ++i + 1,
+                    get content() {
+                        return source.slice(this.start + 1, this.end);
```

Comment:
why do you increment here too?

Reply:
The backslash of an escape doesnt form part of its literal output, so we want to slice with an offset of 1 to exclude it